pkgs/development/ocaml-modules/lem/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, ncurses
+, makeWrapper
+, ocamlbuild
+, findlib
+, ocaml
+, num
+, zarith
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lem";
+  version = "2022-12-10";
+
+  src = fetchFromGitHub {
+    owner = "rems-project";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ZQgcuIVRkJS0KtpzjbO4OPHGg6B0TadWA6XpRir30y8=";
+  };
+
+  minimalOCamlVersion = "4.07";
+
```

Comment:
This is not used (you have to check & `throw` yourself).

Reply:
Here is an example: https://github.com/NixOS/nixpkgs/blob/9313ae3ba50422fb5b916103dcca9d77898264fa/pkgs/development/ocaml-modules/uuidm/default.nix#L3-L4

`minimalOCamlVersion` is handled by `buildDunePackage` only.